pytest.ini

```
@@ -36,6 +36,7 @@ markers =
     hugepages: tests that require nodes with hugepages
     service_mesh: tests that require the service mesh operator to be installed
     jumbo_frame: tests that require network configurations supporting jumbo frames
+    arm: tests that require ARM-based cluster supported
```

```suggestion
    arm64: tests that require ARM-based cluster supported
```

for name consistency

---

done.